src/main/java/Duke/tasks/Event.java

```diff
@@ -0,0 +1,33 @@
+package Duke.tasks;
+
+public class Event extends Task {
+    private String time;
+
+    public Event(String answer) {
```

Comment:
The choice name of variable `answer` does not seem very informative in the context fo your constructor function.

Reply:
Perhaps a better naming might be `userInput`?

Reply:
Sure, I've changed them to userInput